pkg/ctl/cmdutils/configfile.go

```diff
@@ -524,6 +525,12 @@ func NewDeleteNodeGroupLoader(cmd *Cmd, ng *api.NodeGroup, ngFilter *filter.Node
 			return ErrMustBeSet("--name")
 		}
 
+		if flag := l.CobraCommand.Flag("parallel"); flag != nil && flag.Changed {
+			if val, _ := strconv.Atoi(flag.Value.String()); val > 25 || val < 1 {
+				return fmt.Errorf("--parallel value must be of range 1-25")
+			}
```

Comment:
how do I unit test this? I couldn't find an example of mocking out a set flag in the unit tests :/

Reply:
https://github.com/weaveworks/eksctl/pull/4837/files#diff-53b760bebac7b9b835fce483c773f0571f3243541b707f5ae273395779215f03R205